packages/grid/src/HeaderCell.css

```diff
@@ -88,3 +88,7 @@
   cursor: col-resize;
   z-index: 2;
 }
+
+[aria-label="sort column"] {
```

Comment:
Id target this with a class name instead. The label might change or be customisable.